src/functions/overline.js

```diff
@@ -34,9 +34,11 @@ defineFunction({
             positionType: "firstBaseline",
             children: [
                 {type: "elem", elem: innerGroup},
-                {type: "kern", size: 3 * line.height},
-                {type: "elem", elem: line},
+                // The kern on the next line would ordinarily be 3 * line.height
+                // But we put the line into a span that is 5 lines tall, to
+                // overcome a Chrome rendering issue.
```

Comment:
Shouldn't there be a kern that's `3 * line.height - 2 * ruleWidth` or is the change so slight that it's not worth bothering about?

Reply:
I don't see a difference in that distinction. From `buildCommon.makeLineSpan`, we have 
~~~
line.height = options.fontMetrics().defaultRuleThickness;
~~~
So the proposed 2 lines of space in the SVG plus the one-line-high kern sum up to the same distance as the previous `3 * line height`. I'll amplify the comment, but I don't see any other action to take. Am I missing something?

Reply:
I was missed the connection between `lineHeight` and `ruleWidth`.